### DISCUSSION TITLE
Ignoring tracker errors and viewing tracker status

### DISCUSSION THREAD

**Discussion by Stonyx** [jesec/flood#473] [Ideas]
Is there a way to make Flood ignore tracker errors, it's so common for some tracker to be down and then everything just has a red bar making it rather difficult to have a good overview. It's also rather difficult to find out an individual tracker's status, would it be possible to display that too?

This is a revival of the original feature request in the previous/deprecated/archived (???) Flood repo (https://github.com/Flood-UI/flood/issues/486).  It's a feature that would be very handy to have implemented.

**Comment by urwrstkn8mare** [comment]
> It's also rather difficult to find out an individual tracker's status, would it be possible to display that too?

Just for reference: https://github.com/jesec/flood/discussions/527

**Comment by saadbruno** [comment]
> Is there a way to make Flood ignore tracker errors

Maybe not ignore tracker errors. what i'd like to see instead is to only show a torrent as red if **all** trackers have errors. If at least 1 tracker is working,  then we just show it as blue

**Reply by hoffw** [reply]
This is what I'm actively looking for as well, imo it shouldn't be too hard to implement and is a pretty basic feature

**Reply by infradragon** [reply]
i agree, the red error state should only occur if there is a serious problem, not if one of the many trackers didnt repond (very common).

**Comment by trim21** [comment]
the problem is, you can not distinguish if the error comes from tracker or file system in rtorrent

**Reply by infradragon** [reply]
i suppose an alternate solution would to have an intermediate "warning" level that would just put a yellow dot or something on the torrent for minor errors like trackers not responding instead of making the whole thing red with an exclamation mark

**Reply by trim21** [reply]
you still didn't get it. there is no way to know if the error is `minor errors` or `major errors` in rtorrent, there is only single "error" in rtorrent, no filesystem error or tracker error.

**Reply by hoffw** [reply]
I think what @trim21 is saying is that this is less of a Flood issue and more of an rtorrent issue

**Reply by trim21** [reply]
> I think what @trim21 is saying is that this is less of a Flood issue and more of an rtorrent issue

yes, we can impl this for qb, tr and maybe deluge(I'm not sure about this), but we can't impl this for rtorrent

**Reply by infradragon** [reply]
i see what you mean, i had assumed this would be implemented via pattern matching per torrent engine
at the very least there should be an option to suppress errors either globally or via regex patterns

**Comment by trim21** [comment]
I add this feature for tr and qb, you can try latest branch.

and this will need something from rtorrent side, if you need this feature, please send a issue to rtorrent.

**Reply by raidolo** [reply]
@trim21 hi! I can't find the settings you mentioned for qbittorent,I'm using the latest release 4.16.1, doss it include the feature?
